<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <widget class="QGroupBox" name="gb_out">
      <property name="title">
       <string>Out</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="Tree" name="tree_out">
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="out_remove">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="gb_log">
      <property name="title">
       <string>Log</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="Tree" name="tree_log">
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="log_remove">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Comment</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="te_comment"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QGroupBox" name="gb_sweep">
      <property name="title">
       <string>Sweep</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="Tree" name="tree_sw">
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string># pts</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Range</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="sw_remove">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Sweep</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Wait between set and get:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="PyScientificSpinBox" name="sb_before_wait">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="value">
          <double>0.020000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionInstruments"/>
   <addaction name="actionDisplay"/>
   <addaction name="separator"/>
   <addaction name="actionStartSweep"/>
  </widget>
  <action name="actionInstruments">
   <property name="icon">
    <iconset>
     <normaloff>../resources/instruments.svg</normaloff>../resources/instruments.svg</iconset>
   </property>
   <property name="text">
    <string>Instruments</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionDisplay">
   <property name="icon">
    <iconset>
     <normaloff>../resources/display.svg</normaloff>../resources/display.svg</iconset>
   </property>
   <property name="text">
    <string>Display</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionStartSweep">
   <property name="icon">
    <iconset>
     <normaloff>../resources/play.svg</normaloff>../resources/play.svg</iconset>
   </property>
   <property name="text">
    <string>Start sweep</string>
   </property>
   <property name="toolTip">
    <string>Start sweep</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyScientificSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>pyHegel.gui.ScientificSpinBox</header>
  </customwidget>
  <customwidget>
   <class>Tree</class>
   <extends>QTreeWidget</extends>
   <header>widgets/Tree.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
